FreeBSD-8 dropped sgtty.h in favor of termios.h,
without this rlfe build on FreeBSD-8 fails with a 
missing sgtty.h.
See also bug #301508.

--- readline-6.1/examples/rlfe/os.h.orig
+++ readline-6.1/examples/rlfe/os.h
@@ -186,7 +186,7 @@
  *    terminal handling
  */
 
-#ifdef POSIX
+#if defined(POSIX) || defined(__FreeBSD__)
 # include <termios.h>
 # ifdef hpux
 #  include <bsdtty.h>
